### PR TITLE
Fix phpstan errors

### PR DESCRIPTION
**Pull request by MickaelTH**
Attempt to fix phpstan, but still phpunit fail:

PHPUnit 6.5.13 by Sebastian Bergmann and contributors.

...........E..EE........F......FFFFF                              36 / 36 (100%)

Time: 8.16 seconds, Memory: 18.00MB

There were 3 errors:

1) Bref\Test\Bridge\Psr7\RequestFactoryTest::test multipart form data is supported
InvalidArgumentException: Can't find multi-part content

\bref\vendor\riverline\multipart-parser\src\Riverline\MultiPartParser\Part.php:109
\bref\src\Bridge\Psr7\RequestFactory.php:63
\bref\tests\Bridge\Psr7\RequestFactoryTest.php:104

2) Bref\Test\Bridge\Psr7\RequestFactoryTest::test arrays in name are supported with multipart form data
InvalidArgumentException: Can't find multi-part content

\bref\vendor\riverline\multipart-parser\src\Riverline\MultiPartParser\Part.php:109
\bref\src\Bridge\Psr7\RequestFactory.php:63
\bref\tests\Bridge\Psr7\RequestFactoryTest.php:160

3) Bref\Test\Bridge\Psr7\RequestFactoryTest::test files are supported with multipart form data
InvalidArgumentException: Can't find multi-part content

\bref\vendor\riverline\multipart-parser\src\Riverline\MultiPartParser\Part.php:109
\bref\src\Bridge\Psr7\RequestFactory.php:63
\bref\tests\Bridge\Psr7\RequestFactoryTest.php:193

--

There were 6 failures:

1) Bref\Test\CliTest::test deploying requires mandatory files to exist
Failed asserting that exception message 'The command ""\bref\tests/../bref deploy" "\bref\tests/Fixture/EmptyDirectory"" failed.

Exit Code: 1(General error)

Working directory: \bref

Output:
================

Error Output:
================
'"\bref\tests/../bref deploy"' is not recognized as an internal or external command,
operable program or batch file.
' contains 'The files `bref.php` and `serverless.yml` are required to deploy'.

2) Bref\Test\JsHandler\JsHandlerTest::test PHP stdout is forwarded to Node stdout
'"node runner.js \"[]\""' is not recognized as an internal or external command,
operable program or batch file.

Failed asserting that 1 matches expected 0.

\bref\tests\JsHandler\JsHandlerTest.php:82
\bref\tests\JsHandler\JsHandlerTest.php:20

3) Bref\Test\JsHandler\JsHandlerTest::test PHP stderr is forwarded to Node stdout
'"node runner.js \"[]\""' is not recognized as an internal or external command,
operable program or batch file.

Failed asserting that 1 matches expected 0.

\bref\tests\JsHandler\JsHandlerTest.php:82
\bref\tests\JsHandler\JsHandlerTest.php:28

4) Bref\Test\JsHandler\JsHandlerTest::test PHP handler can return a response
'"node runner.js \"[]\""' is not recognized as an internal or external command,
operable program or batch file.

Failed asserting that 1 matches expected 0.

\bref\tests\JsHandler\JsHandlerTest.php:82
\bref\tests\JsHandler\JsHandlerTest.php:36

5) Bref\Test\JsHandler\JsHandlerTest::test PHP handler receives the lambda event
The system cannot find the path specified.

Failed asserting that 1 matches expected 0.

\bref\tests\JsHandler\JsHandlerTest.php:82
\bref\tests\JsHandler\JsHandlerTest.php:48

6) Bref\Test\JsHandler\JsHandlerTest::test PHP errors are forwarded by Node
'"node runner.js \"[]\""' is not recognized as an internal or external command,
operable program or batch file.

Failed asserting that 1 matches expected 0.

\bref\tests\JsHandler\JsHandlerTest.php:82
\bref\tests\JsHandler\JsHandlerTest.php:56

ERRORS!
Tests: 36, Assertions: 53, Errors: 3, Failures: 6.

### CODE REVIEW

File: src/Console/Deployer.php [hunk 1]
```diff
@@ -59,7 +59,7 @@ function ($value, $key) {
             array_keys($parameters)
         ));
 
-        $process = new Process('serverless invoke local ' . $p, '.bref/output');
+        $process = new Process(['serverless invoke local ' . $p, '.bref/output']);
```
Comment (staabm) [inline src/Console/Deployer.php]: I guess this should be

```suggestion
        $process = new Process(['serverless invoke local ' . $p], '.bref/output');
```

File: src/Console/Deployer.php [hunk 2]
```diff
@@ -81,7 +81,7 @@ public function deploy(SymfonyStyle $io, bool $dryRun, ?string $stage): void
             if ($stage !== null) {
                 $serverlessCommand .= ' --stage ' . escapeshellarg($stage);
             }
-            $process = new Process($serverlessCommand, '.bref/output');
+            $process = new Process([$serverlessCommand, '.bref/output']);
```
Comment (staabm) [inline src/Console/Deployer.php]: I guess this should be

```suggestion
            $process = new Process([$serverlessCommand] '.bref/output');
```

File: src/Console/Deployer.php (hunk 1 above)
Comment (mnapoli) [inline src/Console/Deployer.php]: I think the way the new syntax should be with Symfony\Process is something like this:

```php
new Process(['serverless', 'invoke', 'local', $p], '.bref/output'])
```

And I think `$p` should be split too for each argument. No need to `implode()` and escape ourselves the arguments it seems: https://symfony.com/doc/current/components/process.html#using-features-from-the-os-shell

File: src/Console/Deployer.php [hunk 3]
```diff
@@ -203,7 +203,7 @@ private function generateArchive(ProgressBar $progress): void
 
     private function runLocally(string $command): void
     {
-        $process = new Process($command, '.bref/output');
+        $process = new Process([$command, '.bref/output']);
```
Comment (staabm) [inline src/Console/Deployer.php]: just a guess

```suggestion
        $process = new Process([$command], '.bref/output');
```

File: tests/CliTest.php [hunk 4]
```diff
@@ -13,7 +13,7 @@ class CliTest extends TestCase
      */
     public function test deploying requires mandatory files to exist()
     {
-        $process = new Process(__DIR__ . '/../bref deploy', __DIR__ . '/Fixture/EmptyDirectory');
+        $process = new Process([__DIR__ . '/../bref deploy', __DIR__ . '/Fixture/EmptyDirectory']);
```
Comment (staabm) [inline tests/CliTest.php]: you guessed it... ;)

```suggestion
        $process = new Process([__DIR__ . '/../bref deploy'], __DIR__ . '/Fixture/EmptyDirectory');
```

File: tests/JsHandler/JsHandlerTest.php [hunk 5]
```diff
@@ -60,7 +60,7 @@ public function test PHP errors are forwarded by Node()
 
     private function runFile(string $file, array $event = []): Process
     {
-        $process = new Process('node runner.js ' . escapeshellarg(json_encode($event)), __DIR__);
+        $process = new Process(['node runner.js ' . escapeshellarg(json_encode($event)), __DIR__]);
```
Comment (staabm) [inline tests/JsHandler/JsHandlerTest.php]:
```suggestion
        $process = new Process(['node runner.js ' . escapeshellarg(json_encode($event))], __DIR__);
```

**Comment by MickaelTH** [comment]
> Regarding the PHPUnit build it seems your branch is lagging behind master. You can either rebase your branch, recreate it from scratch or merge master in your branch.

Completely stuck on how to provide update on the PR, so not sure how to fix this as there is no option on GitHub for that (I'm a BitBucket user sorry).

**Pull request closed without merging**